pkg/network/protocols/postgres/model_linux.go

```diff
@@ -74,11 +75,10 @@ func (e *EventWrapper) Operation() Operation {
 // extractTableName extracts the table name from the query.
 func (e *EventWrapper) extractTableName() string {
 	fragment := string(e.Tx.getFragment())
-	// Check if the string contains "IF EXISTS",
-	// temp solution for the fact that ObfuscateSQLString does not support "IF EXISTS".
-	if strings.Contains(fragment, "IF EXISTS") {
-		fragment = strings.ReplaceAll(fragment, "IF EXISTS", "")
-	}
+	// Temp solution for the fact that ObfuscateSQLString does not support "IF EXISTS" or "if exists", so we remove
+	// it from the fragment if found.
+	re := regexp.MustCompile("(?i)if exists")
+	fragment = re.ReplaceAllString(fragment, "")
```

Comment:
1. since the regex is not changing with every new call to `extracTableName` then you should define it as a global var
2. Is it truly the optimal solution? for example, what's the cost of `Strings.ToLower()` and then `Strings.Replace` compared to regex?

Reply:
change as we discussed